boards/arm/nrf5340dk_nrf5340/nrf5340_cpunet_reset.c

```diff
@@ -13,6 +13,7 @@
 LOG_MODULE_REGISTER(nrf5340dk_nrf5340_cpuapp, CONFIG_LOG_DEFAULT_LEVEL);
 
 #if defined(CONFIG_BT_CTLR_DEBUG_PINS_CPUAPP)
+#include "soc_secure.h"
```

Comment:
This should be:
```
#include <soc_secure.h>
```
and instead of here, it should be added in that included `debug.h`, right before this line: https://github.com/zephyrproject-rtos/zephyr/blob/a75f3e495cc27e8880033a6a0b86099f905a55a1/subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/debug.h#L36
Then the problem will be solved also for other boards (`bl5340_dvk` and `nrf5340_audio_dk_nrf5340`).